backend/fms_core/services/sample.py

```diff
@@ -359,8 +359,7 @@ def extract_sample(process: Process,
             if concentration_destination is not None:
                 sample_destination_data["concentration"] = concentration_destination
             elif sample_kind_destination.concentration_required:
-                # Force extracted sample to have a concentration if sample_kind is DNA.
-                errors.append(f"Concentration must be specified for an extracted sample if the sample_kind is DNA.")
+                sample_destination_data["concentration"] = 1
```

Comment:
Most likely wrong.

Reply:
I think we'll get rid of the concentration_required attribute on sample_kind ... it was just for DNA and with this change it no longer hold the road. So remove the field on the sample_kind model and remove all tests that use that field. As far as i can see extraction is the only place it's used (a few easy test tweaks too).

Reply:
... and yeah this is wrong...